src/db/index.js

```diff
@@ -1,3 +1,4 @@
+const _ = require('lodash')
```

Comment:
could just grab the cloneDeep function itself

Reply:
word